docs/triggertemplates.md

```diff
@@ -47,9 +47,9 @@ spec:
           type: git
           params:
           - name: revision
-            value: $(params.gitrevision)
+            value: $(tt.params.gitrevision)
           - name: url
-            value: $(params.gitrepositoryurl)
+            value: $(tt.params.gitrepositoryurl)
 ```
 
 `TriggerTemplates` currently support the following [Tekton Pipelines](https://github.com/tektoncd/pipelines) resources:
```

Comment:
We need to update https://github.com/tektoncd/triggers/blob/ff20c8592a1b419ac70b2f46987ded3658bfeada/docs/triggertemplates.md#parameters

Reply:
Updated doc :+1: